.ci/linux-steps.yaml

```diff
@@ -70,7 +70,7 @@ steps:
   displayName: 'Build'
 
 # Run tests via ctest.
-- script: cd build && CTEST_OUTPUT_ON_FAILURE=1 ctest -T Test .
+- script: cd build && CTEST_OUTPUT_ON_FAILURE=1 ctest -j8 -T Test .
```

Comment:
Azure pipelines runs on `Standard_DS2_v2` machines (https://docs.microsoft.com/en-us/azure/virtual-machines/dv2-dsv2-series#dsv2-series) which has two virtual cores, so not sure it makes sense to use more than two here.

Reply:
Agree with @zoq the `-j8`will work, but it will overload the VM, and we will not gain any time.